docs/intermediate-modding/custom-config-syncing.md

```diff
@@ -0,0 +1,233 @@
+
+---
+prev: false
+next: false
+description: An intermediate overview of how to sync custom configs for your Lethal Company mods.
+---
+
+# Custom Config Syncing
+
+::: warning
+**This is an intermediate article. While this introduces some C# concepts, it is recommended to understand C# and the basics of modding this game <i>before</i> reading this article.**
+:::
+
+## Preface
+A very common case for many mod developers is wanting to synchronize the host's configuration file with all other players.
+
+There are many different ways you could achieve this, but we will only go through the two most straight-forward approaches that should work for the majority of cases.
+
+## Automatically Syncing (ConfigurableCompany)
+This is a save-dependent mod that acts as an API, therefore you will need to add it as a dependency if you plan to upload your mod to Thunderstore.
+You can download and learn about it [here](https://thunderstore.io/c/lethal-company/p/AMRV/ConfigurableCompany/) - note that it does not have a **GitHub** repository at the current moment.
```

Comment:
I'm not a fan of promoting / listing any closed-source mods. The wiki has a fairly strong open-source stance.

On a side note, who is the author of this mod?

Reply:
Me neither If I'm honest, but I wanted to include a simpler alternative just in case.
The author is AMRV - I don't know their discord but I've yet to see any open sourced config sync API (at least on GitHub).

If you think the manual section will suffice, removing it may not be the biggest of deals.

Reply:
We may want to remove that for now, until we have the mod/api-specific section in place.

Reply:
I've remove it for now and also fixed the index and info files to pass all checks